docs/src/modules/components/Ad.js

```diff
@@ -212,33 +192,18 @@ function Ad(props) {
     }
 
     const delay = setTimeout(() => {
-      if (!eventLabel) {
-        return;
-      }
-
       window.ga('send', {
         hitType: 'event',
         eventCategory: 'ad',
         eventAction: 'display',
         eventLabel,
       });
-
-      if (eventLabel.indexOf('in-house') === 0) {
-        window.ga('send', {
-          hitType: 'event',
-          eventCategory: 'in-house-ad',
-          eventAction: 'display',
-          eventLabel: children.props.ad.name,
```

Comment:
Now I remember why we had two different events. The objective was to keep track of the performance of each in-house ad (so we could optimize). What about we add this back?

Reply:
Hum, my bad, I needed to look deeper

Reply:
Basically, right now, we don't need to have the display information for each individual ad because we evenly distribute the displays.